tasks/configure.yml

```diff
@@ -35,3 +48,14 @@
     mode: 0644
   with_dict: "{{ prometheus_targets }}"
   when: prometheus_targets != {}
+
+- name: copy prometheus custom static targets
+  copy:
+    src: "{{ item }}"
+    dest: "{{ prometheus_config_dir }}/file_sd/{{ item | splitext | difference(['.yml']) | join('') }}.yml"
```

Comment:
Should we put those custom files in another dir to avoid overwriting files created from `prometheus_targets` or maybe someone has a better idea?